ajax/libs/jquery.mb.bgndgallery/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "jquery.mb.bgndgallery",
+  "filename": "jquery.mb.bgndGallery.min.js",
+  "version": "1.9.5",
+  "description": "This plugin let you create a flexible photo gallery that you can use also as background of your site.",
+  "keywords": [
+    "jquery-plugin",
+    "image",
+    "gallery",
+    "background",
+    "CSS3",
+    "effects",
+    "HTML5"
+  ],
+  "author": "Pupunzi <matteo@open-lab.com> (Matteo Bicocchi)",
+  "license": "MIT",
```

Comment:
Didn't the origin issue and its self and its license file mention GPL?
https://github.com/pupunzi/jquery.mb.bgndGallery/blob/master/licenses/GPL-LICENSE.txt